Modules/Office365DSC/DSCResources/MSFT_TeamsGuestSettings/MSFT_TeamsGuestSettings.schema.mof

```diff
@@ -0,0 +1,10 @@
+[ClassVersion("1.0.0.0"), FriendlyName("TeamsGuestSettings")]
+class MSFT_TeamsGuestSettings : OMI_BaseResource
+{
+    [Key, Description("The Name of the team")] string TeamName;
+    [Write, Description("Allow guests to create and update channels in team ")] Boolean AllowCreateUpdateChannels;
```

Comment:
Remove empty space at the end of Description with a period. This is picky, but this is now used to automatically generate our wiki pages and needs to be clean.

Reply:
done